sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_sender_async.py

```diff
@@ -206,7 +207,9 @@ async def schedule_messages(
                 :caption: Schedule a message to be sent in future
         """
         # pylint: disable=protected-access
+
         self._check_live()
+        messages = create_messages_from_dicts_if_needed(messages, ServiceBusMessage)    # type: ignore
```

Comment:
Why did typing need to be ignored here, but not line 315?

Reply:
It results in the following mypy error:
`Incompatible types in assignment (expression has type "Union[ServiceBusMessage, List[ServiceBusMessage], ServiceBusMessageBatch]", variable has type "Union[ServiceBusMessage, List[ServiceBusMessage]]")`.

Since `schedule_messages` doesn't accept SBMessageBatch, reassigning `messages` to the result of `create_messages_from_dicts_if_needed` (which accepts all 3) leads to this error. `send_messages` doesn't do this because it accepts all 3, so reassigning `message` doesn't lead to incompatible types.